src/models/PartialFormSubmission.php

```diff
@@ -52,25 +58,39 @@ class PartialFormSubmission extends SubmittedForm
      * @var array
      */
     private static $summary_fields = [
-        'ID'            => 'ID',
-        'PartialLink'   => 'Link',
-        'Created'       => 'Created',
-        'LastEdited'    => 'Last Edited',
+        'ID'          => 'ID',
+        'PartialLink' => 'Link',
+        'Created'     => 'Created',
+        'LastEdited'  => 'Last Edited',
     ];
 
     public function getCMSFields()
     {
         /** @var FieldList $fields */
         $fields = parent::getCMSFields();
-        $fields->removeByName(['Values', 'IsSend', 'PartialFields', 'TokenSalt', 'Token', 'UserDefinedFormID', 'Submitter']);
+        $fields->removeByName([
+            'Values',
+            'IsSend',
+            'PartialFields',
+            'TokenSalt',
+            'Token',
+            'UserDefinedFormID',
+            'Submitter'
```

Comment:
Do you want to remove `PartialUploads` as well, since it's merged into `PartialFields`?

Reply:
Yeps, good thing to make sure it's not duplicate (although I didn't see it, better safe than sorry)